src/Agent.Listener/Configuration/FeatureFlagProvider.cs

```diff
@@ -49,7 +49,7 @@ public async Task<FeatureFlag> GetFeatureFlagAsync(IHostContext context, string
             var client = vssConnection.GetClient<FeatureAvailabilityHttpClient>();
             try
             {
-                return await client.GetFeatureFlagByNameAsync(featureFlagName);
+                return await client.GetFeatureFlagByNameAsync(featureFlagName, false);
```

Comment:
just a nit -- but if there is a possibility to pass this as a named argument, it would be IMHO bettter '...(featureFlagName, checkFeatureExists=false)'

Reply:
added